app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <%= render partial: "layouts/head" %>
+  </head>
+  <body class="app-theme">
```

Comment:
How would you feel about moving skip links in the theme to a partial so we can override just that partial in apps that need multiple?

Reply:
Great suggestion. I have a hard time imagining a need to override anything else in this layout.

Reply:
We can open a ticket to add that feature to the theme gem and update this app to use it once available if that makes sense for working towards wrapping these UI features this sprint.

Reply:
I think I'll just update the theme gem as part of this ticket. It shouldn't take long. In the meantime, we can consider whether this is the right approach (i.e., if we want this many skip links, and this set of them).